e2e/test/framework/loadbalancer_suit.go

```diff
@@ -3,6 +3,7 @@ package framework
 import (
```

Comment:
It would be nice to rename this file `loadbalancer_suite.go` in this PR if you don't mind.